templates/data/opensearch/index.html

```diff
@@ -0,0 +1,530 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1nVdoKoPrXw8lsvHEtMichlPNfxYU9Uj1zmKowcUXQ_w/edit
+{% endblock meta_copydoc %}
+
+{% block title %}An enterprise  solution for OpenSearch from Canonical{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}data/opensearch{% endblock %}
+
+{% block meta_description %}
+  Charmed OpenSearch delivers up to 10 years of support and security maintenance for OpenSearch as an integrated, turnkey solution with advanced management features.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <div class="u-hide--small u-vertically-center u-aspect-ratio--16-9 u-align--center"
+               style="margin-top: -55px">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/6deb3921-opensearch-logo.png",
+            alt="Opensearch",
+            width="198",
+            height="186",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+            }}
+          </div>
+        </div>
+        <div class="col-9 col-medium-4">
+          <div class="p-section--shallow">
+            <h1>OpenSearch for the enterprise</h1>
+            <p>
+              Secure and automate the deployment, maintenance and upgrades of your search and analytics suite across private and public clouds.
+            </p>
+          </div>
+          <hr />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="data-spark-modal">Contact us</a>
+          <a class="p-button"
+             href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf">Datasheet</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-text--small-caps highlight">Why choose Canonical's enterprise-grade OpenSearch?</h2>
```

Comment:
No need for this one
```suggestion
        <h2 class="p-text--small-caps">Why choose Canonical's enterprise-grade OpenSearch?</h2>
```